### PR TITLE
Adds table headers for DEX pairs table

### PR DESCRIPTION
**Pull request by mayurbhandary**
## High Level Overview of Change
Adds column headers to DEX Pairs table

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After

Before:
image

After:
image

### CODE REVIEW

File: src/containers/Token/DEXPairs/index.js [hunk 1]
```diff
@@ -138,6 +138,13 @@ const DEXPairs = props => {
       <div className="title"> {t('top_trading_pairs')}</div>
       <div className="pairs-table">
         <table>
+          <thead>
+            <tr>
+              <th className="pair-header">Pair</th>
+              <th>Issuer</th>
+              <th className="stats-header">Stats</th>
```
Comment (mvadari) [inline src/containers/Token/DEXPairs/index.js:145]: Is there a better header we can use than "Stats"? Seems a bit vague to me

Comment (mayurbhandary) [reply src/containers/Token/DEXPairs/index.js:145]: I'm open to suggestions. The component shows the lowest and highest offers currently available. In the future we might add the average exchange rate as a tooltip.

Comment (mvadari) [reply src/containers/Token/DEXPairs/index.js:145]: What about something like "Offer Range"?

Comment (mayurbhandary) [reply src/containers/Token/DEXPairs/index.js:145]: Sure that works for me.

**Comment by mayurbhandary** [comment]
@mvadari, I've included the fix to the error we were getting in this PR. I moved the `/fonts` folder into `src`.

File: src/containers/Token/DEXPairs/index.js (hunk 1 above)
Comment (mayurbhandary) [reply src/containers/Token/DEXPairs/index.js]: Updated in latest

**Review by mvadari — APPROVED** [review]

**Pull request merged**